cache/cache.go

```diff
@@ -329,6 +332,27 @@ func (in *kialiCacheImpl) GetZtunnelPods(cluster string) []v1.Pod {
 	return podList.Items
 }
 
+func (in *kialiCacheImpl) GetZtunnelDaemonset(cluster string) []appsv1.DaemonSet {
+	kubeCache, err := in.GetKubeCache(cluster)
+	if err != nil {
+		in.zl.Debug().Msgf("Unable to get kube cache when checking for ambient profile: %s", err)
```

Comment:
This message duplicates at least one above, I suggest you alter the messages to be unique in some way, so that if we see it we can easily distinguish where it came from.